tests/test_cbv.py

```diff
@@ -56,12 +79,39 @@ def __init__(self, two: int = Depends(dependency_two)):
             def int_dependencies(self) -> int:
                 return self.one + self.two
 
-        client = TestClient(router)
+        return
+
+    def test_dependencies(self, cbv_router: None, client: TestClient) -> None:

```

Comment:
Why would `cbv_router: None` how does it make sense?

Reply:
it returns None the fixture onlt initalizes the router and there is no need to return it as we don't use the object itself in the test